src/EA.Iws.Web/Areas/NotificationApplication/Controllers/ShareNotificationController.cs

```diff
@@ -135,7 +135,7 @@ public async Task<ActionResult> ShareNotification(Guid id, ShareNotificationView
 
                 if (!isInternalUser)
                 {
-                    model = this.PrepareModelWithErrors("Email Address", "Enter a valid email address", model);
+                    model = this.PrepareModelWithErrors("Email Address", "You cannot share this notification with an competant authority user", model);
```

Comment:
Assuming this is from Mike? Could you please update the typo, 'a' rather than 'an' and it should be compet**e**nt.